bundles/org.openhab.binding.meteostick/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -190,6 +198,15 @@
 		</state>
 	</channel-type>
 
+	<channel-type id="rain-today">
+		<item-type>Number:Length</item-type>
+		<label>Rainfall (today)</label>
```

Comment:
```suggestion
		<label>Rainfall Today</label>
```

please also re-run i18n plugin

Reply:
Dear @lsiepel,

I committed the suggested change via the ipad app, but due to the commit message not being according openhab specifics (signing off with name and valid e-mail address) I had to perform a rebase and a reword of the commit message on my Ubuntu PC using the terminal (command line). So far so good.

What I don't understand is your remark "please also re-run i18n plugin". I tried to google and search in the forum to check what I have to do exactly... but to no avail.

Reply:
Documentation can be found here: https://www.openhab.org/docs/developer/utils/i18n.html#generating-i18n-properties-file

It is needed as the cvhange in the XML has to be reflexted in the .properties file.